app/lib/finder_api.rb

```diff
@@ -116,6 +121,12 @@ def allowed_values_for_facet_details(facet_key, facet_details)
       .reject { |f| f["label"].blank? || f["value"].blank? }
   end
 
+  def allowed_values_from_registry(facet_key)
+    registries.all[facet_key].values
+      .map { |_, results| present_facet_option(results, facet_key) }
+      .reject { |f| f["label"].blank? || f["value"].blank? }
```

Comment:
Might be worth doing `.reject { |option| option["slug"].blank? }`  before the `map`, as this should have the same effect as the above `reject`.

Reply:
I don't think that'd have _exactly_ the same effect - `option['title']` could still be populated, but would then be excluded, which doesn't happen in the current setup. This is also the logic that exists in `allowed_values_for_facet_details`.